codeval.py

```diff
@@ -1,42 +1,21 @@
 from canvasapi import Canvas
 from configparser import ConfigParser
-from distutils.dir_util import copy_tree
 import click
 import datetime
 import os, shutil, sys
-import requests
-import zipfile
 import subprocess
 import tempfile
-import time
 import traceback
+from commons import *
```

Comment:
can you expand this import? it's good to be clear on exactly what we are importing from where.

Reply:
Expanded all import *